bindings/python/test.py

```diff
@@ -593,69 +610,70 @@ def test_execute_client(self):
             my_stdin = slave_fd
 
         process = sub.Popen(
-            [sys.executable,"client.py","url_seed_multi.torrent"],
+            [sys.executable, "client.py", "url_seed_multi.torrent"],
             stdin=my_stdin, stdout=sub.PIPE, stderr=sub.PIPE)
         # python2 has no Popen.wait() timeout
         time.sleep(5)
         returncode = process.poll()
-        if returncode == None:
+        if returncode is None:
             # this is an expected use-case
             process.kill()
         err = process.stderr.read().decode("utf-8")
         self.assertEqual('', err, 'process throw errors: \n' + err)
         # check error code if process did unexpected end
-        if returncode != None:
+        if returncode is not None:
             # in case of error return: output stdout if nothing was on stderr
             if returncode != 0:
                 print("stdout:\n" + process.stdout.read().decode("utf-8"))
-            self.assertEqual(returncode, 0, "returncode: " + str(returncode) + "\n"
-                + "stderr: empty\n"
-                + "some configuration does not output errors like missing module members,"
-                + "try to call it manually to get the error message\n")
+            self.assertEqual(returncode, 0, "returncode: " + str(returncode) + "\n" +
+                             "stderr: empty\n" +
+                             "some configuration does not output errors like missing module members," +
+                             "try to call it manually to get the error message\n")
```

Comment:
is there some standard formatting convention that defines this?

Reply:
It looks like we should keep the original based on this: https://lintlyci.github.io/Flake8Rules/rules/W503.html

I guess flake8 has changed but I still got the warning in my linter. Ultimately, I wouldn't really do either but I figured just moving the operator would quiet the linter. This is how I would do it:

```python
self.assertEqual(returncode, 0, ("returncode: {}\n"
"stderr: empty\n"
"some configuration does not output errors like missing module members,"
"try to call it manually to get the error message\n".format(returncode))
```